server/controllers/paragraph.ts

```diff
@@ -0,0 +1,27 @@
+import { Request, Response } from 'express';
```

Comment:
TODO: Think about how we could use it with websockets. Maybe to keep it inside model or helper function instead of controller?